docs/standard/microservices-architecture/container-docker-introduction/docker-defined.md

```diff
@@ -7,49 +7,82 @@ ms.date: 05/26/2017
 ---
 # What is Docker?
 
-[Docker](https://www.docker.com/) is an [open-source project](https://github.com/docker/docker) for automating the deployment of applications as portable, self-sufficient containers that can run on the cloud or on-premises. Docker is also a [company](https://www.docker.com/) that promotes and evolves this technology. Docker works in collaboration with cloud, Linux, and Windows vendors, including Microsoft.
+[Docker](https://www.docker.com/) is an [open-source project](https://github.com/docker/docker) for automating the deployment of applications as portable, self-sufficient containers that can run on the cloud or on-premises. Docker is also a [company](https://www.docker.com/) that promotes and evolves this technology, working in collaboration with cloud, Linux, and Windows vendors, including Microsoft.
 
 ![](./media/image2.png)
```

Comment:
Since you are making a major update, this and all images should have alt text added for accessibility.  For example, this one should be something like:

```
![Docker deploys containers at all layers of the hybrid cloud](./media/image2.png)
```

Reply:
Done.